library/des.c

```diff
@@ -573,6 +599,9 @@ static void des3_set3key( uint32_t esk[96],
                           const unsigned char key[24] )
 {
     int i;
+    DES_VALIDATE( esk != NULL );
```

Comment:
This is a static function. Please remove those checks or justify why they should be done in this function desptie it being static.

Reply:
Thanks, I will remove it.

Reply:
Removed.